### DISCUSSION TITLE
Trying to run the ggml-LLaMa-65B-quantized model

### DISCUSSION THREAD

**Discussion by dumauro** [imartinez/privateGPT#765] [Q&A]
Hi all!

I'm trying to run the program using this model [ggml-LLaMa-65B-quantized](https://huggingface.co/CRD716/ggml-LLaMa-65B-quantized/tree/main)

```
This is my .env
PERSIST_DIRECTORY=db
MODEL_TYPE=LlamaCpp
MODEL_PATH=models/ggml-LLaMa-65B-q4_0.bin
EMBEDDINGS_MODEL_NAME=all-MiniLM-L6-v2
MODEL_N_CTX=1000
MODEL_N_BATCH=8
TARGET_SOURCE_CHUNKS=4
```

But when I run privateGPT.py, this error appears:

> "error loading model: unknown (magic, version) combination: 67676a74, 00000003; is this really a GGML file?
> llama_init_from_file: failed to load model
> Traceback (most recent call last):
>   File "/home/mauro/IA/myai/privateGPT.py", line 83, in <module>
>     main()
>   File "/home/mauro/IA/myai/privateGPT.py", line 36, in main
>     llm = LlamaCpp(model_path=model_path, n_ctx=model_n_ctx, n_batch=model_n_batch, callbacks=callbacks, verbose=False)
>   File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
> pydantic.error_wrappers.ValidationError: 1 validation error for LlamaCpp
> __root__
>   Could not load Llama model from path: models/ggml-LLaMa-65B-q4_0.bin. Received error  (type=value_error)"

**Can anyone guide me if it is possible to work with this model?**

**Comment by Ciaranwuk** [comment]
Have you got it working with other Llama models yet? I usually find I get a similar error to this when my environment isn't right, so might be worth setting yourself up in a new virtual environment and re-installing the requirements

**Reply by dumauro** [reply]
It still hasn't resolved. What is your suggested environment?

**Reply by Ciaranwuk** [reply]
Have you got it working with any smaller models yet?

**Reply by Ciaranwuk** [reply]
in terms of environment, I use mamba to create a new virtual env (plenty of info on how to do that online, it's nice and simple), activate it, then run "pip install -r requirements.txt" again. that makes sure it only has the packages you need in there and reduces the chances of conflicts between dependencies. Have you checked your Python version is correct as well?
